app/javascript/chat/chat.jsx

```diff
@@ -1433,6 +1439,11 @@ export default class Chat extends Component {
     event.currentTarget.classList.remove('opacity-25');
     processImageUpload(files, this.handleImageSuccess, this.handleImageFailure);
   };
+  sendCanvasImage = (files) => {
+    console.log([files]);
```

Comment:
Remove the `console.log`.

Reply:
For Ruby, `binding.pry` statements get picked up by Rubocop and fail the pre-commit hook. Can we do the same for JS?

Reply:
@citizen428, currently console statements get flagged as warnings I believe. We'd just need to update our eslint configuration with the [no-console](https://eslint.org/docs/rules/no-console) rule adjusted to be treated as an error as they shouldn't be hanging around in the code base.

Reply:
@citizen428, issue #10360 has been created to modify the `no-console` eslint rule.

Reply:
Thanks @nickytonline!